src/operators/plus_equal.cpp

```diff
@@ -31,6 +31,68 @@
 #include "../bigint.hpp"
 
 namespace libbig {
+  largeInt &operator += (largeInt a) {
+        largeInt res;
+        string s1 = this.number;
+        string s2 = a.number;
+        res = addLargeInt(s1, s2);
+        return res;
```

Comment:
Codacy found an issue: [Reference to local variable returned.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285288)